apps/web/composables/useEditor/useEditor.ts

```diff
@@ -1 +1,5 @@
-export const useEditor = () => useState('isEditing', () => false);
+export const useEditor = () => {
+  const isEditing = useState('isEditing', () => false);
+  const isEditingDisabled = useState('isEditingDisabled', () => false);
```

Comment:
```suggestion
  const useEditorState = useState('useEditor', () => {
    isEditing = false,
    isEditingDisabled = false
  });
```
Can we combine this into a single state?

Reply:
**suggestion (naming)**: I think `isEditingEnabled` would be preferable to avoid double negatives ("you're not allowed to not edit").